build.sbt

```diff
@@ -0,0 +1,144 @@
+      Some("snapshots" at nexus + "content/repositories/snapshots")
+    else
+      Some("releases" at nexus + "service/local/staging/deploy/maven2")
+  },
+  pomExtra := (
+    <developers>
+      <developer>
+        <id>sjrd</id>
+        <name>Sébastien Doeraene</name>
+        <url>https://github.com/sjrd/</url>
+      </developer>
+      <developer>
+        <id>gzm0</id>
+        <name>Tobias Schlatter</name>
+        <url>https://github.com/gzm0/</url>
+      </developer>
+    </developers>
+  ),
+  pomIncludeRepository := { _ => false },
+
+  // Environments
+
+  jsEnvKind := JSEnvKind.NodeJS,
+
+  Test / jsEnv := {
+    import JSEnvKind._
+
+    val old = (Test / jsEnv).value
+
+    jsEnvKind.value match {
+      case NodeJS =>
+        old
+      case JSDOMNodeJS =>
+        new JSDOMNodeJSEnv()
+      case Firefox =>
+        val profile = new FirefoxProfile()
+        profile.setPreference("privacy.file_unique_origin", false)
+        val options = new FirefoxOptions()
+        options.setProfile(profile)
+        options.setHeadless(true)
+        new SeleniumJSEnv(options)
+      case Chrome =>
+        val options = new ChromeOptions()
+        options.setHeadless(true)
+        options.addArguments("--allow-file-access-from-files")
+        val factory = new DriverFactory {
+          val defaultFactory = SeleniumJSEnv.Config().driverFactory
+          def newInstance(capabilities: org.openqa.selenium.Capabilities): WebDriver = {
+            val driver = defaultFactory.newInstance(capabilities).asInstanceOf[ChromeDriver]
+            driver.manage().timeouts().pageLoadTimeout(1, TimeUnit.HOURS)
+            driver.manage().timeouts().setScriptTimeout(1, TimeUnit.HOURS)
+            driver
+          }
+          def registerDriverProvider(provider: DriverProvider): Unit =
+            defaultFactory.registerDriverProvider(provider)
+        }
+        new SeleniumJSEnv(options, SeleniumJSEnv.Config().withDriverFactory(factory))
+    }
+  },
+))
+
+val commonSettings = Def.settings(
+  // sbt-header configuration
+  headerLicense := Some(HeaderLicense.Custom(
+    s"""scalajs-java-securerandom (${homepage.value.get})
+       |
+       |Copyright EPFL.
+       |
+       |Licensed under Apache License 2.0
+       |(https://www.apache.org/licenses/LICENSE-2.0).
+       |
+       |See the NOTICE file distributed with this work for
+       |additional information regarding copyright ownership.
+       |""".stripMargin
+  )),
+)
+
+lazy val root: Project = project.in(file("."))
+  .enablePlugins(ScalaJSPlugin)
+  .settings(commonSettings: _*)
+  .settings(
+    name := "scalajs-java-securerandom",
+    Compile / packageBin / mappings ~= {
+      _.filter(!_._2.endsWith(".class"))
+    },
+    exportJars := true,
+  )
+
+lazy val testSuite = crossProject(JSPlatform, JVMPlatform)
+  .in(file("test-suite"))
+  .jsConfigure(_.enablePlugins(ScalaJSJUnitPlugin))
+  .enablePlugins(BuildInfoPlugin)
```

Comment:
Side note: We should use this for the core repo as well. Right now, we roll our own :)